ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -6,9 +6,9 @@ syntax = "proto3";
 package com.daml.ledger.api.v1;
 
 import "com/daml/ledger/api/v1/value.proto";
+import "com/daml/ledger/api/v1/commands.proto";
```

Comment:
This is not a "clean" import. `ContractMetadata` must be lifted out of the `commands.proto`